anitab-org/bridge-in-tech-backend
# Issue#86 PUT /user/personal_background api endpoint and tests

### Description
Allow user to update their personal background data

Fixes #86 

### Type of Change:
- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
- Login as a user that has no perrsonal background data and get personal details (GET /user/personal_details) so the user_id is saved to cookie.
- send PUT /user/personal_background request and on successful create, see response 201 Created as per screenshot below
<img width="845" alt="Screen Shot 2020-08-03 at 9 57 43 pm" src="https://user-images.githubusercontent.com/29667122/89181211-92c01180-d5d6-11ea-8c71-faf24bcbd4ac.png">

- now test update by modifying the personal background. On successful update see response 200 OK as per screenshot below 
<img width="829" alt="Screen Shot 2020-08-03 at 9 58 06 pm" src="https://user-images.githubusercontent.com/29667122/89181425-0eba5980-d5d7-11ea-86cc-57901da76ab7.png">

* with invalid new personal background data payload
<img width="844" alt="Screen Shot 2020-08-03 at 10 19 20 pm" src="https://user-images.githubusercontent.com/29667122/89181589-6bb60f80-d5d7-11ea-93a4-a6dc20f49b83.png">

### Checklist:

**Code/Quality Assurance Only**
